iOS_4x_release.md

```diff
@@ -0,0 +1,48 @@
+# Aligning major version releases for the Adobe iOS SDK
+
+### Why are we getting another major update already?
+
+Apple has started doing an annual requirement to increase the minimum version of Xcode used when [submitting to the App Store](https://developer.apple.com/ios/submit/). Each Xcode version only supports back to a specific minimum version of iOS. Loosely speaking, the Xcode version and _minimum_ iOS version are correlated. 
+
+In order to align with Apple's changing requirements, we will also be doing an annual major release of the Adobe Experience Platform Mobile SDKs for iOS. In this yearly major update, you can expect the following changes:
+ - Increasing the minimum supported version of iOS.
+ - Increasing the minimum supported version of Xcode.
+ - Possible increase to minimum supported version of Swift.
+
+### Why are all the extensions updating to major version 4?
+
+The main goal for this change is to simplify Adobe Mobile SDK distribution. Aligning the extensions on the same major version allows us to create predictability. For example:
+ - All Adobe-owned mobile SDK extensions that share a major version are compatible with one another.
+ - Each major version of an Adobe-owned mobile SDK extension has a designated minimum supported OS version.
+
+### Why did some of the extensions skip major versions?
+
+You may be wondering "what happened to versions 2 and 3 of AEPEdge?" 
```

Comment:
Minor nit, but if this a quote, it should be '... be wondering "What happened to...'